packages/connectivity/src/scp-cf/environment-accessor/xsuaa.ts

```diff
@@ -18,7 +18,7 @@ export function getXsuaaServiceCredentials(
     token
   );
   if (!credentials) {
-    throw new Error('Could not find binding to the XSUAA service.');
+    throw new Error(token ? 'Could not find a XSUAA service binding matching the token.' : 'Could not find a XSUAA service binding.');
```

Comment:
[req] Either use "an" or remove the "a". I am in favor of removing.
```suggestion
    throw new Error(token ? 'Could not find XSUAA service binding matching the token.' : 'Could not find XSUAA service binding.');
```